ros_gz_sim_demos/launch/air_pressure.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
```

Comment:
add `.xml` extension

Reply:
Added in [643a69b](https://github.com/gazebosim/ros_gz/pull/632/commits/643a69b9ca2f2630e103d082f90f276f5274c8f0).